### DISCUSSION TITLE
Hỏi về các thông số như: Nhiệt độ trong nhà, Nhiệt độ ngoài trời, Giá trị của cảm biến ánh sáng.

### DISCUSSION THREAD

**Discussion by nguyencharlie** [khoavukhoa3004/DADN_TTNT#8] [Q&A] [answered]
Ask (Device & Sensor): Như mọi người đã biết, thì trong UI của đèn và quạt, còn thể hiện những thông số như Nhiệt độ trong nhà, Nhiệt độ ngoài trời, và Giá trị của cảm biến ánh sáng. Nhưng ở bữa thứ 5 (13/4) thì ở trang Home cũng có những thông số đó. Vậy thì ở UI của đèn và quạt có còn cần thiết nữa không? và nếu cần thì nên để như thế nào cho phù hợp?

**Comment by khoavukhoa3004** [comment, marked as answer]
Tui nghĩ, trang Home sẽ hiển thị nhiệt độ trung bình ngày tính tới thời điểm đó, và hiện trạng thái (Nóng, ẩm, có mưa) theo kiểu người dùng có thể dễ dàng hình dung trực quan. Còn UI của đèn, quạt sẽ hiện chi tiết theo từng giờ với số liệu cụ thể hơn để người dùng nâng cao họ có thể xem chi tiết

**Reply by nguyencharlie** [reply]
Tui nghĩ là trang Home sẽ hiển thị nhiệt độ tức thì ngay tại thời điểm đó, cũng như hiện trạng thái để trực quan.

UI của đèn thì theo tui nghĩ là mình nên bỏ mấy cái thông số đó đi, để chừa khoảng trống đó để hiện lịch sử của thiết bị đèn/quạt.

**Reply by nguyencharlie** [reply]
À mà phần nhiệt độ ở trang Home cũng sẽ nhấn vào được, và giao diện bên trong đó sẽ là toàn bộ lịch sử của các sensor.
